.projenrc.js

```diff
@@ -9,7 +9,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   author: 'Amazon Web Services, Inc.',
   cdkVersion: '2.0.0',
   name: `@aws-cdk/asset-awscli-v${MAJOR_VERSION}`,
-  description: 'An Asset construct that contains the AWS CLI, for use in Lambda Layers',
+  description: 'A library that contains the AWS CLI, as an AssetSource, for use in Lambda Layers',
```

Comment:
Simplify this. "AssetSource" doesn't actually mean anything.
```suggestion
  description: 'A library that contains the AWS CLI for use in Lambda Layers',
```